test_helpers/main.go

```diff
@@ -0,0 +1,238 @@
+package test_helpers
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/tarantool/go-tarantool"
+)
+
+type StartOpts struct {
+	// InitScript is a Lua script for tarantool to run on start.
+	InitScript string
+
+	// Listen is box.cfg listen parameter for tarantool.
+	// Use this address to connect to tarantool after configuration.
+	// https://www.tarantool.io/en/doc/latest/reference/configuration/#cfg-basic-listen
+	Listen string
+
+	// WorkDir is box.cfg work_dir parameter for tarantool.
+	// Specify folder to store tarantool data files.
+	// Folder must be unique for each tarantool process used simultaneously.
+	// https://www.tarantool.io/en/doc/latest/reference/configuration/#confval-work_dir
+	WorkDir string
+
+	// User is a username used to connect to tarantool.
+	// All required grants must be given in InitScript.
+	User string
+
+	// Pass is a password for specified User.
+	Pass string
+
+	// WaitStart is a time to wait before starting to ping tarantool.
+	WaitStart time.Duration
+
+	// ConnectRetry is a count of attempts to ping tarantool.
+	ConnectRetry uint
+
+	// RetryTimeout is a time between tarantool ping retries.
+	RetryTimeout time.Duration
+}
+
+// TarantoolInstance is a data for instance graceful shutdown and cleanup.
+type TarantoolInstance struct {
+	// Cmd is a Tarantool command. Used to kill Tarantool process.
+	Cmd *exec.Cmd
+
+	// WorkDir is a directory with tarantool data. Cleaned up after run.
+	WorkDir string
+}
+
+func isReady(server string, opts *tarantool.Opts) error {
+	var err error
+	var conn *tarantool.Connection
+	var resp *tarantool.Response
+
+	conn, err = tarantool.Connect(server, *opts)
+	if err != nil {
+		return err
+	}
+	if conn == nil {
+		return errors.New("Conn is nil after connect")
+	}
+	defer conn.Close()
+
+	resp, err = conn.Ping()
+	if err != nil {
+		return err
+	}
+	if resp == nil {
+		return errors.New("Response is nil after ping")
+	}
+
+	return nil
+}
+
+var (
+	// Used to extract Tarantool version (major.minor.patch).
+	tarantoolVersionRegexp *regexp.Regexp
+)
+
+func init() {
+	tarantoolVersionRegexp = regexp.MustCompile(`Tarantool (?:Enterprise )?(\d+)\.(\d+)\.(\d+).*`)
+}
+
+// atoiUint64 parses string to uint64.
+func atoiUint64(str string) (uint64, error) {
+	res, err := strconv.ParseUint(str, 10, 32)
```

Comment:
`strconv.ParseUint(str, 10, 64)`

Reply:
Used it here and filed a ticket for cartridge-cli where I did this mistake originally https://github.com/tarantool/cartridge-cli/issues/685